spec/spec_helper.rb

```diff
@@ -26,11 +26,18 @@ def s3
 
 RSpec.configure do |config|
   def converter
-    @converter ||= if !StaticData::JWT_ENABLE
+    @converter ||= if PretestsCheck.jwt_enable?
+                     if StaticData.documentserver_jwt_exist? && StaticData.documentserver_jwt_empty?
+                       OnlyofficeDocumentserverConversionHelper::ConvertFileData.new(StaticData.documentserver_url,
+                                                                                     jwt_key: ENV.fetch('DOCUMENTSERVER_JWT'))
+                     elsif StaticData.jwt_data_exist?
+                       OnlyofficeDocumentserverConversionHelper::ConvertFileData.new(StaticData.documentserver_url,
+                                                                                     jwt_key: StaticData.get_jwt_key.strip)
+                     else
+                       raise 'JWT key is not defined'
+                     end
+                   else
                      OnlyofficeDocumentserverConversionHelper::ConvertFileData.new(StaticData.documentserver_url)
```

Comment:
I think this should be splitted not in one very complicated and multilevel `if` but in several different if with early exit

like this
```
def converter
return @converter if @converter

return @converter =  OnlyofficeDocumentserverConversionHelper::ConvertFileData.new(StaticData.documentserver_url) unless PretestsCheck.jwt_enable?

return @converter = OnlyofficeDocumentserverConversionHelper::ConvertFileData.new(StaticData.documentserver_url,
                                                                                     jwt_key: StaticData.get_jwt_key.strip) if StaticData.jwt_data_exist?
                                                                                     ....
end
```